### PR TITLE
Check the file is present in tar_extract

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by ekohl** [comment]
FYI: I intend to use rspec mocks in all modules and will modulesync that.

File: manifests/tar_extract.pp [hunk 1]
```diff
@@ -8,10 +8,9 @@
 # Actions:
 # - Extracts a tarball
 #
-# Requires:
-# - The certs class
-#
 define certs::tar_extract($path = $title) {
+  validate_file_exists($path)
```
Comment (chris1984) [inline manifests/tar_extract.pp]: Is there a way we can add a message to say, the file does not exist here, and let them know how to regen a certs tar?

**Comment by ekohl** [comment]
I'm leaning to merging this as it is now. Currently it's still a redundant check and the docs are in the installer.

File: manifests/tar_extract.pp (hunk 1 above)
Comment (chris1984) [reply manifests/tar_extract.pp]: +1 was just curious, I don't consider this a blocker.

**Pull request merged**